docs/docsite/rst/vmware/vmware_inventory.rst

```diff
@@ -60,6 +60,61 @@ And you can use this vaulted inventory configuration file using:
 
     $ ansible-inventory -i filename.vmware.yml --list --vault-password-file=/path/to/vault_password_file
 
+Limiting inventory to a datacenter or folder
+============================================
+
+You can limit the search to a specific datacenter or folder. See the example:
```

Comment:
I'm not really clear on what search we are talking about here. Do you want to say something like "You can limit the inventory that a playbook uses by setting the ``search_folder`` to the specific datacenter or subfolder of a datascenter."